certbot-dns-dnsmadeeasy/setup.py

```diff
@@ -8,8 +8,8 @@
 # acme/certbot version.
 install_requires = [
     'acme>=0.21.1',
```

Comment:
This should be updated to match the oldest requirements.

Reply:
Corrected.